consensus/validation.go

```diff
@@ -51,6 +51,16 @@ func (v *defaultMessageValidator) Validate(ctx context.Context, msg *types.Signe
 		return errNonAccountActor
 	}
 
+	if msg.Value.LessThan(types.ZeroAttoFIL) {
```

Comment:
This check is already present here: https://github.com/filecoin-project/go-filecoin/blob/414464d5f086d35fcdcdfb1a7a46e50fbe0e28c6/vm/vm.go#L69

I'm not sure if the check is needed in both places, but I don't think we should be duplicating these consensus rules.

Reply:
@rkowalick Messages should fail faster for these types of errors, so this is appropriate (it will fail the message before it's sent).

@anorth You should use the `IsNegative` method, though.

Reply:
As below, this check happens in both `Sender.send` and `attemptApplyMessage`, so the older version should be removed, since they are redundant.